.travis.yml

```diff
@@ -69,7 +69,7 @@ jobs:
           api_key: $GH_REPO_TOKEN
           on:
             tags: true
-    - d: ldc
+    - d: ldc-1.13.0
```

Comment:
Looks like this started to fail with LDC 1.14: https://github.com/dlang/dub/pull/1661#issuecomment-475217413